README.md

```diff
@@ -981,9 +982,60 @@ Optional configuration:
         onFailure(error);
      }
     ```
-## OpenId Connect
 
-Options for the [OpenId Connect](http://developer.okta.com/docs/api/resources/oidc.html) authentication flow. This flow is required for social authentication, and requires OAuth client registration with Okta. For instructions, see [Social Authentication](http://developer.okta.com/docs/api/resources/social_authentication.html).
+## IdP Discovery
+**:information_source: EA feature:** The Idp Discovery feature is currently an [EA feature](https://developer.okta.com/docs/api/getting_started/releases-at-okta#early-access-ea).
+
+Identity Provider (IdP) Discovery enables you to route users to different 3rd Party IdPs that are connected to your Okta Org. Users can federate back into the primary org after authenticating at the IdP.
```

Comment:
nit: we should define Identity Provider (IdP) wherever the first mention of it appears, which would be in the section title, or in the paragraph above. In the interest of not repeating it in the TOC I would just move that note into the first paragraph of this section.

Reply:
I'd like to move to first paragraph, which will read like
> The Identity Provider (IdP) Discovery feature is currently an ...

thereafter, will always use IdP for short. sounds good?